package.json

```diff
@@ -40,6 +41,7 @@
     "@types/jest": "26.0.0",
     "@typescript-eslint/parser": "^3.0.0",
     "babel-loader": "^8.0.4",
+    "check-engines": "^1.5.0",
```

Comment:
**question:** do we need this?

Reply:
See https://github.com/Financial-Times/dotcom-page-kit/pull/917#discussion_r758259368